modules/org.restlet.ext.platform/src/main/java/org/restlet/ext/platform/Introspector.java

```diff
@@ -225,64 +219,52 @@ public static void process(String[] args) throws TranslationException {
         }
         if (updateCell) {
             if (createNewCell || createNewVersion) {
-                failWithErrorMessage("In update mode, you can't use -d (--create-descriptor), -c (--create-connector) or -N (--new-version). ");
+                failWithErrorMessage("In update mode, you can't use -c (--create-connector) or -N (--new-version). ");
```

Comment:
I guess there is a typo on the "-N" attribute. it should be "-n", right?

Reply:
Bull's eye!